pages/glossary/essential-text-change.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Essential Text Change
+key: essential-text-change
+unambiguous: true
+objective: false
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An _essential text change_ occurs when the modification of the text content of an HTML element is necessary to convey accurate and timely information to the user. An _essential text change_ may appear in any of the following forms:
+
+1. Providing context - Text is changed to provide users with additional information required to understand the current state of the page. For example, while searching for results, text may change to convey the page is *loading* to provide context to the user.
+2. Live updates of important information - The content automatically updates to show the most recent information available from a data source and should not be ignored by the user. For example, updates to a users financial information or browsing sessions status would constitute an `essential-text-change`. 
```

Comment:
Maybe adding something like live news updates in case of danger (e.g. safety/evacuation instructions in case of natural disaster or terror attack).

Reply:
Agreed (and very relevant given the hurricanes hitting the US right now), I will add something to that effect.